recipes/Diamond/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+cd src
+./install-boost &> /dev/null
```

Comment:
Does boost actually end up being packaged in the binary? From looking at that script, it doesn't seem to be copied to $PREFIX.

However, the cmake build instructions do seem to do that well, so maybe they should be used? https://github.com/bbuchfink/diamond#compiling-using-cmake